addons/point_of_sale/static/src/app/store/models.js

```diff
@@ -1380,7 +1380,7 @@ export class Order extends PosModel {
         this.partner = partner;
 
         this.temporary = false; // FIXME
-        this.to_invoice = false; // FIXME
+        this.to_invoice = json.to_invoice;
```

Comment:
I'm not sure that the to_invoice is always in the json. If yes, then that is ok, if not then maybe put it  this way `this.to_invoice = json.to_invoice ? json.to_invoice : false;`